### ISSUE TITLE
about PseudoConv3d

### ISSUE THREAD

**Issue by Weifeng-Chen** [bryandlee/Tune-A-Video#5]
I've seen your definition of PseudoConv3d, it seems that you use conv2D+temporal1D to replace the conv3d( in VDM, it convert 3x3 2Dconv to 1x3x3 3Dconv), which can inherit from the 2d weights very fast, Am I right? AND are there any other differences between the one you implement and the paper claim?

**Comment by bryandlee** [comment]
Hi, you are correct. 2+1 conv is from make-a-video paper, and initialized as an identity function. The default settings won't update the temporal conv weights, so the model is actually same as the one used in tune-a-video paper. The learning rate & train steps are different from the paper.

**Comment by Weifeng-Chen** [comment]
【自动恢复】来信已收到，我将尽快回复你！

**Comment by Weifeng-Chen** [comment]
> 

okay, thanks!
